lib/tasks/check_khousheh.rake

```diff
@@ -285,8 +294,16 @@ namespace :check do
                     cluster_title = cluster_center == pm.id ? pm.title : cluster.title
                     updated_cluster_attributes[:title] = cluster_title
                     # Update cluster
-                    # FIXME: Update clusters in batches
-                    cluster.update_columns(updated_cluster_attributes)
+                    cluster_items[cluster.id] = updated_cluster_attributes
+                  end
+                end
+                # Bulk-update Cluster
+                unless cluster_items.blank?
+                  begin
+                    cluster_items_values = cluster_items.values.to_a
+                    Cluster.upsert_all(cluster_items_values, unique_by: :id)
```

Comment:
@melsawy , this won't work unless you change also how `updated_cluster_attributes` is calculated... between lines 266 - 277 (on `develop`) you can see that `updated_cluster_attributes` are calculated based on the current `cluster` instance, which used to be updated at every iteration... now that you're planning to bulk-update clusters, you need to change these lines to read from the updated cluster instance in memory, not from the database. Makes sense? Let's talk if it's not clear.